### PR TITLE
Made changes to DNS section and minor typo fixes

### PR DESCRIPTION
**Pull request by garrett-wood**
Properly double-spaced some lists so they appear correct
Corrected X509 to X.509
Added warning letting users know to point DNS records to primary AD FS server when using a WID deployment before switching the DNS records to the production Load Balancer value

### CODE REVIEW

**Comment by illfated** [comment]
Oh. I see. Some lists don't wrap in MarkDown if they don't have 2 blank spaces at end-of-line. I was not aware of this issue.

I wonder if they would have wrapped correctly if those sub-lists were indented with 4 blank spaces instead of only 3.

File: WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md [hunk 1]
```diff
@@ -208,6 +208,9 @@ In order to effectively balance the traffic, the ILB should be configured with l
 Using your internal DNS server, create an A record for the ILB. The A record should be for the federation service with the IP address pointing to the IP address of the ILB. For example, if the ILB IP address is 10.3.0.8 and the federation service installed is fs.contoso.com, then create an A record for fs.contoso.com pointing to 10.3.0.8.
 This will ensure that all data trasmitted to fs.contoso.com end up at the ILB and are appropriately routed. 
 
+> [!WARNING]
+>If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md:212]: Codestyle/consistency: may I ask you to add the following MarkDown indent marker compatibility spacing?
```suggestion
> If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.
```

File: WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md [hunk 2]
```diff
@@ -208,6 +208,9 @@ In order to effectively balance the traffic, the ILB should be configured with l
...
+
 > [!NOTE]
 >If your deployment is also using IPv6, be sure to create a corresponding AAAA record.
 >
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md:216]: Same here, codestyle/consistency (MarkDown indent marker compatibility spacing)
```suggestion
> If your deployment is also using IPv6, be sure to create a corresponding AAAA record.
```
(also removing redundant blank line)

File: WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md [hunk 3]
```diff
@@ -218,6 +221,11 @@ This will ensure that all data trasmitted to fs.contoso.com end up at the ILB an
 
 In order to ensure that Web Application Proxy servers are able to reach the AD FS servers behind the ILB, create a record in the %systemroot%\system32\drivers\etc\hosts for the ILB. Note that the distinguished name (DN) should be the federation service name, for example fs.contoso.com. And the IP entry should be that of the ILB's IP address (10.3.0.8 as in the example).
 
+> [!WARNING]
+>If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.
+>
+>
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md:227]: As above, codestyle/consistency.
```suggestion
> If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.

```
(plus removal of redundant blank line and redundant MD indent markers)

**Comment by garrett-wood** [comment]
> Oh. I see. Some lists don't wrap in MarkDown if they don't have 2 blank spaces at end-of-line. I was not aware of this issue.
> 
> I wonder if they would have wrapped correctly if those sub-lists were indented with 4 blank spaces instead of only 3.

So far as I'm aware it's a requirement in markdown to either double space after the end of a line to be a continuation (eg. Shift-Enter in a work processor) or double enter for a new paragraph.

File: WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md [hunk 4]
```diff
@@ -291,7 +297,7 @@ Overall, you need the following rules to efficiently secure your internal subnet
 ![EXT access rules (inbound)](./media/how-to-connect-fed-azure-adfs/nsg_dmz.png)
 
 > [!NOTE]
-> If client user certificate authentication (clientTLS authentication using X509 user certificates) is required, then AD FS requires TCP port 49443 be enabled for inbound access.
+> If client user certificate authentication (clientTLS authentication using X.509 user certificates) is required, then AD FS requires TCP port 49443 be enabled for inbound access.
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md:300]: (minor grammatical detail in the original text)
```suggestion
> If client user certificate authentication (clientTLS authentication using X.509 user certificates) is required, then AD FS requires TCP port 49443 to be enabled for inbound access.
```

File: WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md [hunk 5]
```diff
@@ -208,16 +208,22 @@ In order to effectively balance the traffic, the ILB should be configured with l
...
+> If your deployment is also using IPv6, be sure to create a corresponding AAAA record.
 >
 
 ### 7. Configuring the Web Application Proxy server
 **7.1. Configuring the Web Application Proxy servers to reach AD FS servers**
 
 In order to ensure that Web Application Proxy servers are able to reach the AD FS servers behind the ILB, create a record in the %systemroot%\system32\drivers\etc\hosts for the ILB. Note that the distinguished name (DN) should be the federation service name, for example fs.contoso.com. And the IP entry should be that of the ILB's IP address (10.3.0.8 as in the example).
 
+> [!WARNING]
+> If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md:224]: Adding a comma here should make it easier to keep 2 meanings separated:
```suggestion
> If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server, or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.
```

**Comment by illfated** [comment]
> So far as I'm aware it's a requirement in markdown to either double space after the end of a line to be a continuation (eg. Shift-Enter in a work processor) or double enter for a new paragraph.

Fair enough. From my own personal experience, it also works if the indentation is well defined. Then again, the method you have chosen is valid and working too.

**Comment by illfated** [comment]
Looks quite good so far. I will add my approval when you have committed the suggested changes.

**Comment by illfated** [comment]
#sign-off

**Comment by billmath** [comment]
#sign-off

**Pull request merged**